### PR TITLE
Adding 100 days of machine learning playlist

### LINKED ISSUE #1151: [Add] Machine Learning Youtube playlist
### Site URL

https://www.youtube.com/playlist?list=PLKnIA16_Rmvbr7zKYQuBfsVkjoLcJgxHH

### Description of the above provided site

100 days Machine Learning playlist
mll

### Category

Youtube

### Sub Category

machine learning

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Pro-18**
closes #1151

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
please don't commit changes of your previous PR. Also, work on one issue at a time and don't forget to checkout our contributing guidelines. Thanks

**Review by k-deepak04 — APPROVED** [review]

**Comment by k-deepak04** [comment]
@CBID2 please review this and merge it

File: database/languages/python.json [hunk 1]
```diff
@@ -33,5 +33,13 @@
         "url": "https://realpython.com/",
         "category": "languages",
         "subcategory": "python"
+    },
+    {
+        "name": "Complete Python Tutorial",
+        "description": "Live examples will help you learn Python. This tutorial covers file handling, various modules, Matplotlib, machine learning, MongoDB, and MySQL.",
```
Comment (CBID2) [inline database/languages/python.json:39]: [Proofreading]
```suggestion
        "description": "This Python tutorial covers file handling, various modules, Matplotlib, machine learning, MongoDB, and MySQL.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Pro-18! :) Just made a suggestion for improvement

**Comment by CBID2** [comment]
@Pro-18, please resolve the merge conflicts in your branch

**Comment by rupali-codes** [comment]
@Pro-18 are you still willing to work on this issue?

**Pull request closed without merging**